src/FakeItEasy/Expressions/ExpressionArgumentConstraintFactory.cs

```diff
@@ -77,7 +80,7 @@ private static object InvokeExpression(Expression expression)
             return Expression.Lambda(expression).Compile().DynamicInvoke();
         }
 
-        private IArgumentConstraint GetArgumentConstraintFromExpression(Expression expression)
+        private IArgumentConstraint GetArgumentConstraintFromExpression(Expression expression, out object value)

```

Comment:
Well, I'd use a tuple, but we can't use C# 7 yet :wink:
We could use `Tuple<IArgumentConstraint, object>`, but IMO it's worse than `out`.
Creating a class just for this seems a bit overkill to me...